data/human/deep jobs.txt

```diff
@@ -353,3 +353,616 @@ mission "Record academic conference"
+		conversation
+			`As you walk away from a food stand in the spaceport a rather young man in active wear steps away from the wall and smoothly sidles up next to you between one step and the next.`
+			`	"Hello Captain. Please don't be alarmed, do you have a moment?"`
+			`	His words come out very quickly as if they'd been heavily rehearsed and, looking around, no one appears to be paying any attention to either of you.`
+			choice
+				`	"Sure. Who are you?"`
+					goto who
+				`	"Sorry, but I'm busy right now."`
+			`	"Uh, I was coming to thank you if that helps?" he says.`
+			choice
+				`	"Oh. What is it?"`
+				`	"Sorry, maybe next time."`
+					decline
+			
+			label who
+			`	He continues walking and speaks without looking directly at you. "My name is Wayde, Agent Wayde if you will. Don't mind the title though, I'm uh, well I work in the logistics division for Deep Security. You've transported a large number of secure packages recently and several of them were mine."`
+			choice
+				`	"Work is work."`
+					goto blindacceptance
+				`	"Ah, yes."`
+					goto blindacceptance
+				`	"Do all the secure packages belong to Deep Security?"`
+			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs.`
+			label blindacceptance
+			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion and it seems like you qualified. You passed the background check at any rate.`
+			`	"So at any rate, I'm authorized to give you a one time offer to join that list. Don't worry, I'm not asking anything of you right now."`
+			`	Belatedly he seems to do a double-check that no one is actively listening to the conversation as you walk. "So, you interested?"`
+			choice
+				`	"Oh sure, I'm willing to help."`
+					goto agree
+				`	"Sorry, but I don't think I want to get that involved."`
+					decline
+				`	"You said 'at any rate' twice."`
+			`	He looks away sheepishly. "Er, yes, sorry about that. The question stands though."`
+			choice
+				`	"Oh sure, I'm willing to help."`
+					goto agree
+				`	"Sorry, but I don't think I want to get that involved."`
+					decline
+			label agree
+			`	"Fantastic," he says, suddenly grinning widely enough to see in your periphery. "I'll add you to the list as soon as I get back to the office again. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`
+			apply
+				set "deep security reveal"
+			`	Agent Wayde jogs off ahead of you, flicking a subtle salute your direction just before he turns a corner and disappears.`
```

Comment:
He's young and you're a captain, just for fun, as a formal wave, enthusiasm for salutes, respect, out of habit... like really take your pick.

Also, any motion can be subtle if it aligns sufficiently well with another action you'd be expected to do, so that only the person it's aimed at is likely to take notice.
e.g. A jogging person might push their hair back off their face, if you're not looking right at them a quick salute could be made a part of that action and it would not jump out as conspicuous to someone who was merely walking by and not explicitly watching the jogger.